engine/src/main/scala/cromwell/engine/workflow/package.scala

```diff
@@ -0,0 +1,35 @@
+package cromwell.engine
+
+import akka.actor.{ActorSystem, ActorRef}
+import com.typesafe.scalalogging.StrictLogging
+import cromwell.services.MetadataServiceActor.{MetadataServiceAction, PutMetadataAction, MetadataServiceResponse, MetadataEvent}
+
+import scala.concurrent.{Future, ExecutionContext}
+
+package object workflow {
+  // Sent when there is a failure to push to the metadata service
+  private[workflow] case class MetadataServiceFailureResponse(reason: Throwable)
+  private[workflow] case class MetadataActionFailureResponse(reason: Throwable, action: MetadataServiceAction)
+
+  object WorkflowMetadataKeys {
+    val Name = "workflowName"
+    val Id = "id"
+    val SubmissionTime = "submission"
+    val StartTime = "start"
+    val EndTime = "end"
+    val Status = "status"
+    val Outputs = "outputs"
+    val Inputs = "inputs"
+    val Calls = "calls"
+  }
+
+  implicit class EnhancedServiceRegistry(val serviceRegistryActor: ActorRef) extends AnyVal {
```

Comment:
@Horneth , But making it util sounds like a common behavior for other Actors/Service, is that the intention?